File: 1th grade/Listas de Exercícios (Back-end [SPOLOGP])/C/Lista 1 - Casa/6.c
#include <stdio.h>
#include <math.h>

int main ()
{
    float r, r_2, A, C;
    
    printf("Área e comprimento de uma circunferência\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");

    printf("Raio da circunferência: "); scanf("%f", &r);
    
    r_2=pow(r, 2);

    A=3.14*r_2;
    C=2*3.14*r;
    
    printf("\nÁrea: %.2f(medida)²\nComprimento: %.2f(medida)\n", A, C);
}

File: 1th grade/Listas de Exercícios (Back-end [SPOLOGP])/Python/Lista 1 - Casa/6.py
print("Área e comprimento de uma circunferência\n")

print("--Valores decimais devem ser digitados com (.)!--")

r=float(input("Raio da circunferência: "))

A=3.14*r**2; A="{:.2f}".format(A)
C=2*3.14*r; C="{:.2f}".format(C)

print(f"\nÁrea: {A}(medida)²\nComprimento: {C}(medida)\n")